hackerank/js-practice/time-conversion.js

```diff
@@ -0,0 +1,40 @@
+function timeConversion(string) {
+  const containPM = string.includes("PM");
+  let oldTime = string.slice(0, 2);
+  let newTime = parseInt(oldTime);
+  if (containPM) {
```

Comment:
I suggest extracting this to a function like PMTimeConversion and the else statement to AMTimeConversion to have cleaner code.